### ISSUE TITLE
Since this project is no longer being developed, can you at least release the unreleased APKs somewhere?

### ISSUE THREAD

**Issue by omgwtfbbqpls** [otakuhqz/ZimLX#84]

**Comment by otakuhqz** [comment]
this is the last compiled apk. I have not tested this file
[Zim Launcher-aosp-zim-withQuickstep-release.zip](https://github.com/otakuhqz/ZimLX/files/4848279/Zim.Launcher-aosp-zim-withQuickstep-release.zip).

PD:
a new project is being developed to replace Zim Launcher. I will start sharing the code here in GitHub and the initial releas too.

**Comment by omgwtfbbqpls** [comment]
Thank you. Is this one missing any features from the last official release?

**Comment by otakuhqz** [comment]
No, what I posted is the last build with all the features developed at that date.

The new project already began and some features will be migrated, you will have news in the new project repository
